KProcessHacker/verify.c

```diff
@@ -642,51 +663,56 @@ NTSTATUS KpiVerifyProcess(
         (fileHeaders.OptionalHeader.LoaderFlags != processHeaders.OptionalHeader.LoaderFlags))
     {
         status = STATUS_ACCESS_DENIED;
-        goto Exit;
+        goto CleanupExit;
     }
 
     sizeOfSectionHeaders = sizeof(IMAGE_SECTION_HEADER) * fileHeaders.FileHeader.NumberOfSections;
     if (sizeOfSectionHeaders == 0)
     {
         status = STATUS_ACCESS_DENIED;
-        goto Exit;
+        goto CleanupExit;
     }
-    sectionHeaders = ExAllocatePoolWithTag(PagedPool,
-                                           sizeOfSectionHeaders,
-                                           'pvpK');
-    if (!sectionHeaders)
+
+    if (!(sectionHeaders = ExAllocatePoolZero(PagedPool, sizeOfSectionHeaders, 'pvpK')))
     {
-        goto Exit;
+        status = STATUS_ACCESS_DENIED;
```

Comment:
This should be consistent with the above `ExAllocatePoolZero` ([line 398](https://github.com/processhacker/processhacker/pull/1132/files#diff-6c9dc9ccbf75e62ca944a08a1582237d348db77324191bc00eec3fc9bdc06c8eR398)). My preference would be `STATUS_INSUFFICIENT_RESOURCES` since this is an allocation failure and it's best to communicate that as such. There are other areas in this file that return `STATUS_INSUFFICIENT_RESOURCES` too.